src/sections/Route.enum.ts

```diff
@@ -5,6 +5,7 @@ export enum Route {
   LOG_OUT = '/',
   DATASETS = '/datasets',
   CREATE_DATASET = '/datasets/create',
+  EDIT_DATASET_FILES = '/datasets/edit-files',
```

Comment:
Yes, I wondered the same thing, when looking at the JSF version I saw this url:
<img width="558" alt="Screenshot 2024-04-19 at 09 09 03" src="https://github.com/IQSS/dataverse-frontend/assets/64560524/4a1112da-552e-429f-88f6-26d29276289d">

When you say all links for editing files, which ones do you mean? Maybe I am missing something about how is done in the JSF version.

Reply:
@GermanSaracca I meant the options for the Edit Files dropdown here, but I see now that only the Metadata option goes to another page, the rest of the options are popups.  So there are only two actions that go to another page, Upload Files and Edit Metadata.  I could see how these could go to the same page in different modes, like in JSF, but also it could be implemented as going to two different pages. What do you think?
<img width="174" alt="Screenshot 2024-04-19 at 9 23 08 AM" src="https://github.com/IQSS/dataverse-frontend/assets/675224/1f6bec1c-fb04-4372-b23b-0bc06883c42e">

Reply:
I see, maybe we could change `'/datasets/edit-files'` url for `'/datasets/files'`, and keep that page for uploading files when mode=UPLOAD query param is present (`'/datasets/files?mode=UPLOAD'`) and edit page something like mode=EDIT query param is present together with a selectedFiles query param (`'/datasets/files?mode=EDIT&selectedFiles=26137%2C26139'`).

In this way 👆 , we will  have one page for both editing and uploading. 

Although I think creating two different urls like datasets/edit-files & datasets/upload-files might be cleaner, better for testing and have nicer/prettier urls.

I think I am now more in favor of the second option mentioned above, what are your thoughts about it?

Reply:
I like the second option too

Reply:
I will make the changes to go on that direction 🚀